src/Symfony/Component/Serializer/CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ CHANGELOG
  * Add `CamelCaseToSnakeCaseNameConverter::REQUIRE_SNAKE_CASE_PROPERTIES` context option
  * Deprecate `AbstractNormalizerContextBuilder::withDefaultContructorArguments(?array $defaultContructorArguments)`, use `withDefaultConstructorArguments(?array $defaultConstructorArguments)` instead (note the missing `s` character in Contructor word in deprecated method)
  * Add `XmlEncoder::CDATA_WRAPPING_PATTERN` context option
+ * Support subclasses of DateTime(Immutable) for denormalization
```

Comment:
```suggestion
 * Support subclasses of \DateTime and \DateTimeImmutable for denormalization
```
Please add ticks around the classes, thanks

Reply:
Why do you think they're necessary? 🤔

Reply:
Not necessary, but at least to me its readability is much better, see the lines before 🤷‍♂️

Reply:
```suggestion
 * Support subclasses of `\DateTime` and `\DateTimeImmutable` for denormalization
```
Was on a phone before, adding ticks seems impossible there :-P

Reply:
@OskarStark thanks!

Reply:
> Was on a phone before, adding ticks seems impossible there :-P 

I can do `backticks` on my iPhone. ✌️

Reply:
Because you are a hacker 🤪